src/funcCoreTools/validateFuncCoreToolsInstalled.ts

```diff
@@ -44,9 +48,29 @@ export async function validateFuncCoreToolsInstalled(context: IActionContext, me
                 items.push(DialogResponses.learnMore);
             }
 
+            if (os.platform() === 'linux') {
```

Comment:
Could you extract this out to its own function?

Something like `generateLinuxrrorMessage`? It could return the string that should be appended to the `failedInstall` string.

It's just a lot of code in the middle of the validation that makes it harder to follow the flow of logic imo